src/service/fixed-layer.js

```
@@ -462,6 +476,21 @@ export class FixedLayer {
     }
   }
 
+  /**
+   * Sanitizes an element, cleaning up the style field attribute.
+   * @param {!Element} element
+   * @return {boolean} True if the element has been sanitized and the style
+   *     field has been removed.
+   * @private
+   */
+  sanitizeElement_(element) {
```

Let's throw the user error in this method.